tests/models/test_dag.py

```diff
@@ -1863,13 +1900,29 @@ def test_max_active_runs_not_none(self):
             next_dagrun_create_after=None,
             is_active=True,
         )
+        # assert max_active_runs updated
+        assert orm_dag.max_active_runs == 16
         session.add(orm_dag)
         session.flush()
-
         assert orm_dag.max_active_runs is not None
 
-        session.rollback()
-        session.close()
+    def test_dagmodel_has_import_error_is_false(self):
+        dag = DAG(dag_id='test_dag', start_date=timezone.datetime(2020, 1, 1))
+        DummyOperator(task_id='dummy', dag=dag, owner='airflow')
+
+        session = settings.Session()
+        orm_dag = DagModel(
+            dag_id=dag.dag_id,
+            next_dagrun=None,
+            next_dagrun_create_after=None,
+            is_active=True,
+        )
+        # assert has_import_error is false
+        assert not orm_dag.has_import_errors
+        session.add(orm_dag)
+        session.flush()
+
+        assert not orm_dag.has_import_errors
```

Comment:
This test isn't really adding much -- it's testing `self.has_import_errors = False` in the constructor.

It certainly doesn't need a whole separate test case for this, it can be added as an assert somewhere else if we want it. You already test basically everything here in `test_dags_needing_dagruns_doesnot_send_dagmodel_with_import_errors`, so this whole test function isn't needed